README.md

```diff
@@ -144,7 +169,7 @@ _FieldDB_ is actively worked on by the [https://github.com/OpenSourceFieldlingui
 * [UsageDashboard](https://github.com/OpenSourceFieldlinguistics/UsageDashboard) A realtime updating usage dashboard for fielddb servers (forked from dashing) (_Fork_)
 
 ## Modules
-
+_the term "modules" means something in Python. What does it mean here?_
```

Comment:
Basically, on the branch you committed this (master), edit the file to reflect the changes I suggested. Then:

``` sh
git add -A
git commit --amend
``` 
Edit the commit log if you like. :wq should quit if vi is your native editor. Finally: `git push -f origin master`. This will bundle the changes into your previous commit, and force update this PR.